pkg/cmd/diff/cmd.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2024 The KitOps Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package diff
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"sync"
+
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/spf13/cobra"
+	"oras.land/oras-go/v2/registry"
+
+	"kitops/pkg/cmd/options"
+	"kitops/pkg/lib/constants"
+	"kitops/pkg/output"
+)
+
+const (
+	// Constants for identifying reference types
+	remotePrefix = "remote://"
+	localPrefix  = "local://"
+	//Constants for formatting output tables.
+	layerTableHeadings = "Type    | Digest             | Size"
+	layerTableFormat   = "%-7s | %-18s | %s\n"
+)
+
+type diffOptions struct {
+	options.NetworkOptions
+	configHome string
+	refA       *registry.Reference
+	refB       *registry.Reference
+}
+
+func DiffCommand() *cobra.Command {
+	opts := &diffOptions{}
+	cmd := &cobra.Command{
+		Use:   "diff",
+		Short: "Compare two ModelKits",
+		Args:  cobra.ExactArgs(2),
+		Long: `Compare two ModelKits to see the differences in their layers.
+		
+		ModelKits can be specified from either a local or from a remote registry.
+		To specify a local ModelKit, prefix the reference with 'local://', e.g. 'local://jozu.ml/foo/bar'.
+		To specify a remote ModelKit, prefix the reference with 'remote://', e.g. 'remote://jozu.ml/foo/bar'.
+		If no prefix is specified, the local registry will be checked first.
+		`,
```

Comment:
Annoyingly, using backticks preserves strings _verbatim_, including the leading whitespace :) 

```
kit diff --help
Compare two ModelKits

Description:
  Compare two ModelKits to see the differences in their layers.
  		
  		ModelKits can be specified from either a local or from a remote registry.
  		To specify a local ModelKit, prefix the reference with 'local://', e.g. 'local://jozu.ml/foo/bar'.
  		To specify a remote ModelKit, prefix the reference with 'remote://', e.g. 'remote://jozu.ml/foo/bar'.
  		If no prefix is specified, the local registry will be checked first.
  		

Usage:

```

Reply:
It is not just that Use and examples are incorrect too. I completely forgot to update. Fixed now